charts/karpenter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: karpenter
 description: A Helm chart for Karpenter, an open-source node provisioning project built for Kubernetes.
 type: application
-version: 1.4.0
```

Comment:
This will be autogenerated when we cut a patch release, we can drop the version bump here.